lua/init.lua

```diff
@@ -78,8 +78,19 @@ for _,global_variable_setting in ipairs(args.global_variable_settings) do
 end
 
 for _,option_setting in ipairs(args.option_settings) do
+    -- Ignore initial values of lines and columns because they are set by neovim directly.
+    -- See https://github.com/neovide/neovide/issues/2300
+    if option_setting ~= "lines" and option_setting ~= "columns" then
+        vim.api.nvim_create_autocmd({ "VimEnter" }, {
```

Comment:
Perhaps it would make sense to move `VimEnter` outside the loop, And have an internal loop for notifying all settings instead?

Reply:
Yup will do that